src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -20,6 +22,7 @@
     public boolean isHeaderHidden;
     public StudentAttributes studentToViewPageAs;
     public InstructorAttributes previewInstructor;    
+    private String registerMessage;
```

Comment:
I see, will you be working on those too?
Either way, it may be a good idea to put a to-do comment in those files to whoever will be handling it to tidy this up as well when they're working on those.